helm/external-dns-app/templates/deployment.yaml

```
@@ -22,6 +22,9 @@ spec:
         giantswarm.io/service-type: "managed"
         app: {{ .Values.name }}
       annotations:
+        {{- if eq .Values.provider "aws" }}
+        iam.amazonaws.com/role: {{ .Values.clusterID}}-Route53Manager-Role
```

```suggestion
        iam.amazonaws.com/role: {{ .Values.role}}
```
What if we do it more universal so we can reuse for something else?
(adding cluster id with the statis text on value)

---

We don't have role anywhere. We have `clusterID` from here https://github.com/giantswarm/cluster-operator/pull/731 
We could define value in template, but I don't consider this app to be reusable anywhere else for now

---

Also we're using `clusterID` in `kiam` already.

I'll add an issue to wrap it in a different way as a next step

---

https://github.com/giantswarm/giantswarm/issues/7397

---

ok